### DISCUSSION TITLE
[Menubar] Open on-click always

### DISCUSSION THREAD

**Discussion by mbaldas** [radix-ui/primitives#2256] [Help] [answered]
Hello everyone, the default behavior from `Menubar`is to open on hover if there's one menu already open.
If every menu is closed the action to open it for the first time is by clicking it.

How can I achieve the behavior of always needing to click to open the menu? Basically **disabling the hover.**

**Comment by joaom00** [comment, marked as answer]
You can add `onPointerEnter={event => event.preventDefault()}` on `Trigger` https://codesandbox.io/p/sandbox/awesome-lederberg-stgr35?file=%2FApp.jsx%3A16%2C53-16%2C101
